packages/terra-form-checkbox/tests/jest/__snapshots__/CheckboxField.test.jsx.snap

```diff
@@ -567,6 +567,72 @@ exports[`should render an optional part on the label 1`] = `
 />
 `;
 
+exports[`should render checkbox field with div element for safari browser or Edg browser 1`] = `
+<checkboxField
+  intl={
+    Object {
+      "defaultFormats": Object {},
+      "defaultLocale": "en",
+      "formatDate": [Function],
+      "formatHTMLMessage": [Function],
+      "formatMessage": [Function],
+      "formatNumber": [Function],
+      "formatPlural": [Function],
+      "formatRelative": [Function],
+      "formatTime": [Function],
+      "formats": Object {},
+      "formatters": Object {
+        "getDateTimeFormat": [Function],
+        "getMessageFormat": [Function],
+        "getNumberFormat": [Function],
+        "getPluralFormat": [Function],
+        "getRelativeFormat": [Function],
+      },
+      "locale": "en",
+      "messages": null,
+      "now": [Function],
+      "onError": [Function],
+      "textComponent": "span",
+      "timeZone": null,
+    }
+  }
+  legend="Coustom Message CheckboxField"
+/>
+`;
+
```

Comment:
These snapshots look incorrect; you'll need to use a `.dive()` statement to get the inner contents of the rendered components.

Reply:
i have made changes as per suggestions.